src/test/java/org/arquillian/container/chameleon/ConfigurationTestCase.java

```diff
@@ -80,4 +80,22 @@ public void shouldFailOnMissingContainerFile() throws Exception {
         configuration.setChameleonTarget("wildfly:8.2.0.Final:managed");
         configuration.validate();
     }
+
+    @Test
+    public void shouldReturnFalseForMissingContainerName() throws Exception {
+        boolean isSupported = ChameleonConfiguration.isSupported("MISSING_TARGET:8.2.0.Final:managed");
+        Assert.assertFalse(isSupported);
+    }
+
+    @Test
+    public void shouldReturnTrueForGivenContainerName() throws Exception {
+        boolean isSupported = ChameleonConfiguration.isSupported("wildfly:8.2.0.Final:managed");
```

Comment:
but what I see is that you need to translate `glassfish-embedded-3.1` to the chameleon coordinate. You mean that in arquillian addon json file  you'll have this translation right?